### PR TITLE
Feature/95 admin dashboard homepage

### PR DESCRIPTION
**Pull request by enginehouse**

### CODE REVIEW

File: notify-api-key [hunk 1]
```diff
@@ -0,0 +1 @@
+cx_joel_uksa_test-63998e44-a39f-4f58-b2a7-bb8f7a4269e1-95cd417d-1541-437a-b82d-747be5f47c26
```
Comment (Koded) [inline notify-api-key]: Keep keys etc out of the repo

File: static/ui/scss/govuk-template.scss [hunk 2]
```diff
@@ -310,10 +310,7 @@ button:focus {
     line-height: 1;
     text-decoration: none;
     text-rendering: optimizeLegibility;
-    border-bottom: 1px solid transparent;
```
Comment (Koded) [inline static/ui/scss/govuk-template.scss:313]: We don't really want to be changing library files. Is there some other way of overriding this?

File: app/server/api/static/Find Results [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Searching 77 files for "isArray"
```
Comment (Koded) [inline app/server/api/static/Find Results]: Looks like this was committed by accident

**Review by Koded — CHANGES_REQUESTED** [review]
Hi @enginehouse can you merge in develop?

File: static/ui/scss/govuk-template.scss (hunk 2 above)
Comment (enginehouse) [reply static/ui/scss/govuk-template.scss:313]: I didn't intentionally make this change - it is a generated css file added as a sass file for convenience. Please review if you think it requires it

File: app/server/api/static/Find Results (hunk 3 above)
Comment (enginehouse) [reply app/server/api/static/Find Results]: Removed

File: notify-api-key (hunk 1 above)
Comment (enginehouse) [reply notify-api-key]: Removed - please delete this reference if you can : )

Comment (dhilton) [reply notify-api-key]: Can we regenerate the key as well? It's been tainted as it's now in the public domain.

Comment (Koded) [reply notify-api-key]: Just revoked the key on Notify.

**Review by Koded — DISMISSED** [review]

**Review by Koded — APPROVED** [review]

**Pull request merged**